src/sentry/runner/commands/cleanup.py

```diff
@@ -98,22 +99,26 @@ def multiprocess_worker(task_queue: _WorkQueue) -> None:
             return
 
         model_name, chunk = j
-        model = import_string(model_name)
-        try:
-            task = deletions.get(
-                model=model,
-                query={"id__in": chunk},
-                skip_models=skip_models,
-                transaction_id=uuid4().hex,
-            )
 
-            while True:
-                if not task.chunk(apply_filter=True):
-                    break
-        except Exception:
-            logger.exception("Error in multiprocess_worker.")
-        finally:
-            task_queue.task_done()
+        with sentry_sdk.start_transaction(op="cleanup", name="multiprocess_worker") as transaction:
```

Comment:
All the errors from the processes are missing the transaction:

<img width="103" height="115" alt="Image" src="https://github.com/user-attachments/assets/f4dc6d38-78eb-47c5-8832-b6e11c9ad390" />